src/esp/physics/PhysicsManager.cpp

```diff
@@ -48,7 +48,8 @@ bool PhysicsManager::addScene(
 }
 
 int PhysicsManager::addObject(const int objectLibIndex,
-                              DrawableGroup* drawables) {
+                              DrawableGroup* drawables,
```

Comment:
We should probably change these parameters to a `ObjectConfig` of some type, especially since this will soon also include light setup, material setup, etc., some of which we'll want defaults for, others required, and so on... Don't want to have massive parameter lists like we do in `ResourceManager` create drawable pipeline.

Reply:
👍 Good point. We'll continue to discuss this for a later refactor.